lib/active_record/mysql/enum/mysql_adapter.rb

```diff
@@ -25,19 +27,35 @@ def native_database_types #:nodoc
         # be done on a per adapter basis, but is generalized here.
         #
         # will generate enum('a', 'b', 'c') for :limit => [:a, :b, :c]
-        def type_to_sql(type, limit: nil, precision: nil, scale: nil, unsigned: nil, **) # :nodoc:
-          if type.to_s == 'enum'
-            native = native_database_types[type]
-            column_type_sql = (native || {})[:name] || 'enum'
+        if Rails::VERSION::MAJOR < 5
+          def type_to_sql(type, limit = nil, precision = nil, scale = nil, unsigned = nil, **) # :nodoc:
```

Comment:
Maybe `**_options` to be clear that those options are being ignored?

Reply:
done